Repositories.md

```diff
@@ -0,0 +1,29 @@
+With the Kiota onboarding, our code will be using the following four solutions:
+- Kiota generator
+- Kiota core libraries
+- Graph core library which will be dependending on the Kiota core.
+- Graph service library containing Kiota generated the request builders.
+
+The proposed plan to set the GitHub repositories for the above solutions is as follows : 
+
+- Kiota (the generator) - common for all languages
+	
+- kiota-<language>
+	
+- msgraph-sdk-<language>-core
+	
+- msgraph-sdk-<language>
+	
+- msgraph-beta-sdk-<language>
+	
+- (optional) msgraph-china-sdk-<language>
+	
+- (optional) msgraph-beta-china-sdk-<language>
+ 
+Concerns to be discussed more:
+  
+1. Duplicate names - Consider adding new repositories for all the Kiota generated libraries and discontinuing the repos that use the current code generation process.
```

Comment:
in that case I suggest we rename the repo whenever we release to align with the other languages

Reply:
```suggestion
1. Duplicate names - Consider adding new repositories for all the Kiota generated libraries and discontinuing the repos that use the current code generation process. Also consider renaming the older repository.
```